pkg/validation/internal/bundle.go

```diff
@@ -117,6 +126,30 @@ func getOwnedCustomResourceDefintionKeys(csv *operatorsv1alpha1.ClusterServiceVe
 	return keys
 }
 
+// validateBundleSize will raise an error if the bundle size is bigger than the value allowed
+// or a warning when the value is closed to it
+func validateBundleSize(bundle *manifests.Bundle) []errors.Error {
+	warnPercent := 0.15
+	warnSize := int64(max_bundle_sized_allowed - math.Round(max_bundle_sized_allowed*warnPercent))
+	var errs []errors.Error
+
+	if bundle.Size == nil || *bundle.Size == 0 {
+		errs = append(errs, errors.WarnFailedValidation("unable to check the bundle size", nil))
```

Comment:
Should we really raise an error in this case?
It shows safer a warning because we do not want getting started to block/fail in validation because in some edge cases where the loader was not called.

Reply:
I think so